js/foundation.abide.js

```diff
@@ -696,7 +696,7 @@ Abide.defaults = {
    */
   validators: {
     equalTo: function (el, required, parent) {
-      return $(`#${el.attr('data-equalto')}`).val() === el.val();
+      return $(`#${el.attr('data-equalto')}`).get(0).value === el.get(0).value;
```

Comment:
Should use `el.getAttribute()` instead of `el.attr`

Reply:
Please check the title of this PR, it's WIP ;-)